carma/launch/carma_src.launch.py

```diff
@@ -197,7 +197,7 @@ def generate_launch_description():
     is_cp_mot_enabled = LaunchConfiguration('is_cp_mot_enabled')
     declare_is_cp_mot_enabled = DeclareLaunchArgument(
         name='is_cp_mot_enabled',
-        default_value = 'False',
+        default_value = 'True',
```

Comment:
Is MOT being enabled so that the CP pipeline will publish, which the PR description states is required?

Reply:
This is largely due to previous implementation design where we expected external objects to be always published with other msgs like PSM. And having one subscriber's intention is to synchronize all timestamps into one.